README.md

```diff
@@ -189,14 +203,28 @@ Singularity> make MPI=1 MPI_HOME=/container/ompi
 Singularity> exit
 ```
 
+##### SS Containerized Libraries
+
+An example command for running the all_reduce test on two nodes using
+four GPUs per node and the SS libraries built into the container:
+
+```
+$> cd nccl-tests
+$> srun --exclusive -c 72 '--distribution=*:block' --mpi=pmix_v4 -n 512 --ntasks-per-node=4 --cpu-bind=socket --ntasks-per-socket=1 --sockets-per-node=4 singularity run --nv --bind $TMPDIR --bind $HOME --bind `pwd` /path/to/sif/pytorch-ngc-hpc-dev-0050511.sif ./build/all_reduce_perf -b 1M -e 8G -f 2 -n 100 >& out-n$SLURM_NTASKS-ppn4.txt
```

Comment:
This is very minor, but may be make this "pytorch-ngc-hpc-dev-0050511.sif" to some generic name? but leaving it is just fine too.